packages/dialog/src/dialog.ts

```diff
@@ -102,7 +102,7 @@ export class Dialog extends LitElement {
                       `
                     : html``}
             </div>
-            <div class="content">
+            <div class="content" tabindex="0">
```

Comment:
@jnurthen would love some additional insight here. This is based on the feedback in https://dequeuniversity.com/rules/axe/3.4/scrollable-region-focusable?application=axeAPI. However, this element won't always be "scrollable" even though it has `overflow: auto`, due to content size. Do we need to dynamically cover for that? Or, is it OK to make it focusable even when the content doesn't require scroll?

Reply:
@andrewhatran I've thought about this some more, and while it seems sub-optimal to have or not have the `tabindex` 100% of the time like this, it seems slightly less so to have it 100% of the time _AND_ any updates we make to when it is available in the future would essentially be within the private API scope of the element, so there is no need to block on them now. I've created https://github.com/adobe/spectrum-web-components/issues/723 to track the additional functionality and design application that will likely be needed. Without directly addressing this question now, I think we should be OK moving forward on a more complete review of this PR when you have some time! Thanks.

Reply:
This totally makes sense to me. If we're planning on updating this component in the future to make the tabindex responsive to the scrollability of the element, then I feel comfortable with moving forward with this workaround 👍 

I'll continue with a more thorough/complete review as soon as I have time!